vector/src/main/java/im/vector/util/PreferencesManager.java

```diff
@@ -85,6 +85,7 @@
     public static final String SETTINGS_ENCRYPTION_INFORMATION_DEVICE_KEY_PREFERENCE_KEY = "SETTINGS_ENCRYPTION_INFORMATION_DEVICE_KEY_PREFERENCE_KEY";
 
     public static final String SETTINGS_DISPLAY_NAME_PREFERENCE_KEY = "SETTINGS_DISPLAY_NAME_PREFERENCE_KEY";
+    public static final String SETTINGS_DISPLAY_EMAIL_PREFERENCE_KEY = "SETTINGS_DISPLAY_EMAIL_PREFERENCE_KEY";
```

Comment:
Is it normal that this new constant is not used in this PR?

Reply:
No I took it off